### DISCUSSION TITLE
XSLT Transformation query

### DISCUSSION THREAD

**Discussion by ajtruckle** [MicrosoftEdge/WebView2Feedback#3669] [Q&A]
Code:

`<xsl:variable name="Translations" select="document('Group--Field%20Service%20Groups-Translations.xml')"/>`

The above works. This fails:

`<xsl:variable name="Translations" select="document('Group--Field Service Groups-Translations.xml')"/>`

I don't know if the issue is WebView2 or Chromium. Other XSLT processors (like Saxon) can cope with spaces in file names.

Linked StackOverflow:

https://stackoverflow.com/q/76773553/2287576

**Comment by novac42** [comment]
@ajtruckle the string in document() is taken as URI so I suppose the space characters need to be URL encoded. Some XSLT processors might be more lenient but I guess not the one Chromium use. I'm going to move this one to discussion board to see if community would provide more input.

**Comment by ajtruckle** [comment]
Thanks for moving it here @novac42 .
